README.md

```diff
@@ -167,6 +167,30 @@ The ID3 supported frames supported are (see the enum `FrameName`):
 * `.TrackPosition`, track position frame
 * `.Genre`, the genre frame
 * `.AttachedPicture(_ pictureType: ID3PictureType)` the attached picture frame
+* `Comment`
+* `Composer`
+* `Conductor`
+* `ContentGrouping`
+* `Copyright`
+* `DiscPosition`
+* `EncodedBy`
+* `EncoderSettings`
+* `FileType`
+* `FileOwner`
+* `Grouping`
+* `Language`
+* `Lyricist`
+* `Media Type`
+* `MixArtist`
+* `MovementName`
+* `MovementIndex` (aka movement number)
+* `PocastCategory`
+* `PodcastDescription`
+* `PodcastID`
+* `PodcastKeyword`
+* `Publisher`
+* `Subtitle`
+* `UnsyncedLyrics`
 
 Only the `version` field is mandatory. The other fields are optional.
 The field `artist`,  `albumArtist`, `title` and `album` are encoded/saved using Unicode 16 bit string (as requested by specification). 
```

Comment:
You can add a `Contributor` section to the readme with a link to your profile github, linkedin etc. Your contribution will be really big with this pr so it is right that you get your acknowledgment 🍾

Reply:
Honestly, all I did was cut/paste your code, for the most part. As you can see with my attempt to create the IPLS frame, or even modify the comment/language frames, I don't actually understand what I'm doing beyond some basic monkey-see/monkey-do.

Reply:
Also, I noticed when I look at the tags in MediaInfo that the Lyrics tag comes out in another language, one of Asian origin though I'm not sure which one. I suspect that, like the Comment frame, this one is missing some compliance standards.

However, you have an open feature request specifically for this frame from another person so I don't feel right omitting it the way I did the Comment frame. Would you like me to leave it in, or remove it?

Reply:
For the lyrics and MediaInfo we can keep your implementation just to give to the user of the library the frame (even if it is not stricly as described in the standard, I saw other libraries in other language do an implementation similar to your one). When I will be free I will implement all the id3 standard stuff required by this tags.

Reply:
Okay I will leave that as-is for now, then!